ddtrace/tracer/transport_test.go

```diff
@@ -198,17 +198,26 @@ func TestTraceCountHeader(t *testing.T) {
 }
 
 type recordingRoundTripper struct {
-	reqs   []*http.Request
-	client *http.Client
+	reqs []*http.Request
+	//client *http.Client
```

Comment:
nit: do we need this commented out http.Client still?

Reply:
Doh!

Reply:
```suggestion 
```